XenCert/XenCertCommon.py

```diff
@@ -34,6 +34,11 @@
     ["server",          "server name/IP addr", " : ", None,        "required", "-n", ""   ],
     ["serverpath",      "exported path", " : ", None,        "required", "-e", ""     ] ]
 
+__cifs_args__ = [
```

Comment:
Could we have more intuitive switches? Perhaps -r for (share) root,  -p for password. Finding something for username is tough - we could make do with -y.